scripts/update-headers.js

```diff
@@ -0,0 +1,31 @@
+const { createWriteStream } = require('fs');
+const { Readable } = require('stream');
+const { finished } = require('stream/promises');
+const { resolve } = require('path');
+
+const files = ['js_native_api_types.h', 'js_native_api.h', 'node_api_types.h', 'node_api.h'];
+
+const commit = process.argv[2] ?? `${process.version.substring(0,3)}.x`;
```

Comment:
Not really sure what to put here as the default: `main` or the current node executable's major version?

Reply:
Updated to get latest release from https://nodejs.org/download/release/index.json